subprojects/core/src/integTest/groovy/org/gradle/api/ConfigurationOnDemandIntegrationTest.groovy

```diff
@@ -39,7 +40,10 @@ class ConfigurationOnDemandIntegrationTest extends AbstractIntegrationSpec {
         file("gradle.properties") << "org.gradle.configureondemand=true"
     }
 
-    @Requires(value = IntegTestPreconditions.NotParallelExecutor, reason = "parallel mode hides incubating message")
+    @Requires(
+        value = [IntegTestPreconditions.NotParallelExecutor, IntegTestPreconditions.NotIsolatedProjects],
```

Comment:
❓ Since _Isolated Projects_ implies _Configuration Cache_ implies _parallel execution_, shouldn't `NotParallelExecutor` be enough here?

Reply:
Technically speaking, CC executor doesn't imply parallel executor (and was never implying). See
https://github.com/gradle/gradle/blob/3c1f32c7fe1c3bd21eb926d63eae8c4f2045ffd5/testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/GradleContextualExecuter.java#L86

CC <-> parallelism relation is being expressed like a 
https://github.com/gradle/gradle/blob/6a39fc751fc9b3db47e95f669a3d25d773768319/testing/internal-integ-testing/src/main/groovy/org/gradle/test/preconditions/IntegTestPreconditions.groovy#L112

Reply:
Interesting. Thanks for the pointers, @6hundreds. I'm now thinking, why shouldn't we update [this definition](https://github.com/gradle/gradle/blob/3c1f32c7fe1c3bd21eb926d63eae8c4f2045ffd5/testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/GradleContextualExecuter.java#L42) to `configCache(true, true)`?

Reply:
@bamboo It could make sense, yes. Let's do that in separate PR, we still can improve test infrastructure for IP, like a https://github.com/gradle/gradle/issues/29221

Reply:
> Let's do that in separate PR

Oh, yeah, didn't mean otherwise 👍